src/main/java/proj/kedabra/billsnap/utils/ErrorMessageEnum.java

```diff
@@ -35,6 +35,7 @@
     WRONG_TIP_FORMAT("Tip format is incorrect."),
     WRONG_INVITATION_STATUS("The invitation is not in {} status."),
     TAX_ID_DOES_NOT_EXIST("No tax exists with that id: {}"),
+    INCORRECT_LN_FN_INPUT("FirstName and LastName cannot be empty."),
```

Comment:
Why do you need this? 

Technically, you could just have this verification done as NotBlank in the resource?

Reply:
you're right